examples/ffi/exporter.cpp

```diff
@@ -140,10 +140,11 @@ int main(int argc, char *argv[]) {
   ddog_CharSlice info_example = DDOG_CHARSLICE_C_BARE(
       "{\"application\": {\"start_time\": \"2024-01-24T11:17:22+0000\"}, \"platform\": {\"kernel\": \"Darwin Kernel 22.5.0\"}}");
 
+  ddog_prof_Exporter_set_timeout(exporter, 30000);
```

Comment:
I wonder if we even want this?  Tho as an example, maybe its good to show off the new API